### PR TITLE
Add methods to convert single sentence to input tensors

### PR DESCRIPTION
**Pull request by tedhtchang**
* Convert single sentence to input tensor
* UI to submit the sentence
* Debugger for Chrome configuration
* Update readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,19 +20,19 @@ Sentiment Analysis using BERT model and Tensorflowjs
 * Place ./model folder under public directory
 
 ### How to run this app
-1. npm run dev
+1.
+```
```
Comment (pvaneck) [inline README.md]: Don't put this section as a code block. Instead, remove the backticks and only backtick the code portions only like:

Run `npm install` first to install the required modules, then run `npm run dev`.

File: README.md [hunk 2]
```diff
@@ -20,19 +20,19 @@ Sentiment Analysis using BERT model and Tensorflowjs
...
-   0.5241742134094238,
-   0.4758257567882538
-   ]
-   predict: 2085
+   Loaded Tokenizer.
+   105 Model Loading time (ms): 4941
    ```
 ### Convert vocab.txt to vocab.json
-*
+* A included vocabulary file, vocab-1.json, is extracted and converted from "[Bert-base, uncase](https://github.com/google-research/bert#pre-trained-models)". If you would like to convert other pre-trained vocabulary use the followwing script:
```
Comment (pvaneck) [inline README.md]: nit: An* included
nit: pre-trained vocabularies* 
following*

File: README.md (hunk 1 above)
Comment (tedhtchang) [reply README.md]: Thanks. Looks better!

File: README.md (hunk 2 above)
Comment (tedhtchang) [reply README.md]: Changed

**Review by yhwang — APPROVED** [review]

**Pull request merged**